src/osemgrep/networking/Rule_fetching.ml

```diff
@@ -6,6 +6,7 @@ module FT = File_type
 module C = Rules_config
 module R = Rule
 module XP = Xpattern
+module Http_helpers = Http_helpers.Make (Lwt_wrapper)
```

Comment:
maybe call it Lwt_platform instead of Lwt_wrapper

Reply:
Like the name, will do